.github/workflows/build.yml

```diff
@@ -13,3 +13,24 @@ jobs:
       target-name: 'dd_pprof' # target name in binding.gyp
       package-manager: 'npm' # npm or yarn
       cache: true # enable caching of dependencies based on lockfile
+
+  publish:
```

Comment:
I'd rename this to something like dev_publish so the name is clearer in the actions list.